### PR TITLE
docs(docs/operations) Adding more steps to etcd-manager restore operation to work properly

### PR DESCRIPTION
**Pull request by phspagiari**
This PR adds more steps to properly restore an etcd cluster using etcd-manager.

We faced this problem yesterday. When we do the restore, the backup brings back the old master endpoints. I thought the flag `--endpoint-reconciler-type` should get rid of it but doesn't.

In order to bring back the stability of our cluster we had to take these actions after the restore.

### CODE REVIEW

**Comment by rifelpet** [comment]
Thank you, doc improvements are always appreciated!
/ok-to-test
/area documentation

File: docs/operations/etcd_backup_restore_encryption.md [hunk 1]
```diff
@@ -55,22 +55,45 @@ etcd-manager-ctl --backup-store=s3://my.clusters/test.my.clusters/backups/etcd/e
...
+
+A new etcd cluster will be created and the backup will be
+restored onto this new cluster. Please note that this process might take a short while,
+depending on the size of your cluster.
 
 You can follow the progress by reading the etcd logs (`/var/log/etcd(-events).log`)
 on the master that is the leader of the cluster (you can find this out by checking the etcd logs on all masters).
 Note that the leader might be different for the `main` and `events` clusters.
 
+After the restore, you will probably face an intermitent connection to apiserver.
```
Comment (rifelpet) [inline docs/operations/etcd_backup_restore_encryption.md]: Looks like the Travis CI job caught this

```suggestion
After the restore, you will probably face an intermittent connection to apiserver.
```

Comment (phspagiari) [reply docs/operations/etcd_backup_restore_encryption.md]: Done!

**Pull request merged**